Sources/SwiftSyntaxBuilder/MemberAccessExprConvenienceInitializers.swift

```diff
@@ -0,0 +1,30 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+extension MemberAccessExpr {
+    /// Creates a `MemberAccessExpr` using the provided parameters.
+    /// - Parameters:
+    ///   - base:
+    ///   - dot:
+    ///   - name:
+    ///   - declNameArguments:
```

Comment:
To keep this PR focussed, do you think we could pull this change out into a separate PR? Also, I think you’re missing descriptions for the parameters. I think we should either remove the per-parameter documentation altogether or provide descriptions.

Reply:
Fixed in https://github.com/apple/swift-syntax/pull/450